aiohttp/web_reqrep.py

```diff
@@ -576,6 +590,20 @@ def _start_pre_check(self, request):
         else:
             return None
 
+    def _start_compression(self, request):
+        def start(coding):
+            self.headers[hdrs.CONTENT_ENCODING] = coding.name
+            self._resp_impl.add_compression_filter(coding.name)
+
+        if self._compression_force:
+            start(self._compression_force)
+        else:
+            accept_encoding = request.headers.get(hdrs.ACCEPT_ENCODING, '')
+            for coding in ContentCoding:
+                if coding.name in accept_encoding:
+                    start(coding)
+                    return
```

Comment:
If we added full support for content negotiation on this header we would also want to support q values.